cabby/data/wikidata/query.py

```diff
@@ -178,10 +223,10 @@ def get_geofenced_wikidata_items(region: Text) -> Dict:
     return query_api(query)
 
 
-def query_api(query: Text) -> Dict[Sequence, Dict]:
-    '''Get Wikidata query from . 
+def query_api(queries: Sequence[Text]) -> Dict[Sequence, Dict]:
```

Comment:
Both Sequence and Dict are parameterized types. Can you describe in text (in reply here, not in the code yet) what exactly we find in each? (We may want to create data classes for these.)

Reply:
What are the head labels? Can you give an example?

Can you also give an example of a result item? (key and value)

Reply:
head lables - {'vars': ['place', 'placeLabel', 'wikipediaUrl', 'point']}
results- {'place': {'type': 'uri', 'value': 'http://www.wikidata....y/Q3272426'}, 'placeLabel': {'type': 'literal', 'value': 'Equitable Life Building', 'xml:lang': 'en'}, 'point': {'type': 'literal', 'value': 'Point(-74.010555555 ...708333333)'}, 'wikipediaUrl': {'type': 'uri', 'value': 'https://en.wikipedia...Manhattan)'}}

Reply:
Great - please add this to the docstring. I have another branch where I convert these into entity objects and will refine the types and description, and it would have helped to have had this when I was sorting that out. :-)